lab_5_scraper/scraper.py

```diff
@@ -118,6 +218,9 @@ def __init__(self, config: Config) -> None:
         Args:
             config (Config): Configuration
         """
+        self.config = config
+        self.urls = []
+        self.id_counter = 1
```

Comment:
this counter does not need to be an attribute. you do not need it. redesign if you depend on it